torch/ao/quantization/qconfig.py

```diff
@@ -294,13 +297,13 @@ def get_default_qconfig(backend='fbgemm', version=0):
 default_embedding_qat_qconfig_4bit = QConfig(activation=NoopObserver.with_args(dtype=torch.float32),
                                              weight=default_embedding_fake_quant_4bit)
 
-def get_default_qat_qconfig(backend='fbgemm', version=1):
+def get_default_qat_qconfig(backend='x86', version=1):
```

Comment:
can we split changing the default into a separate PR?

Reply:
OK. It's reverted now.